plugins/woocommerce/includes/blocks/class-wc-brands-block-templates.php

```diff
@@ -1,156 +0,0 @@
-<?php
-
-declare( strict_types = 1);
-
-//phpcs:disable Squiz.Classes.ClassFileName.NoMatch, Squiz.Classes.ValidClassName.NotCamelCaps
-/**
- * Utils for compatibility with WooCommerce Full Site Editor Blocks
- *
- * Important: For internal use only by the Automattic\WooCommerce\Internal\Brands package.
- *
- * @version 9.4.0
- */
-class WC_Brands_Block_Templates {
-
-	/**
-	 * Constructor.
-	 */
-	public function __construct() {
-		add_action( 'get_block_templates', array( $this, 'get_block_templates' ), 10, 3 );
-		add_filter( 'get_block_file_template', array( $this, 'get_block_file_template' ), 10, 3 );
-		add_filter( 'woocommerce_has_block_template', array( $this, 'has_block_template' ), 10, 2 );
-	}
-
-	/**
-	 * Get the taxonomy-product_brand template from DB in case a user customized it in FSE
-	 *
-	 * @return WP_Post|null The taxonomy-product_brand
-	 */
-	private function get_product_brand_template_db() {
-		$posts = get_posts(
-			array(
-				'name'           => 'taxonomy-product_brand',
-				'post_type'      => 'wp_template',
-				'post_status'    => 'publish',
-				'posts_per_page' => 1,
-			)
-		);
-
-		if ( count( $posts ) ) {
-			return $posts[0];
-		}
-
-		return null;
-	}
-
-	/**
-	 * Fixes a bug regarding taxonomies and FSE.
-	 * Without this, the system will always load archive-product.php version instead of taxonomy_product_brand.html
-	 * it will show a deprecation error if that happens.
-	 *
-	 * Triggered by woocommerce_has_block_template filter
-	 *
-	 * @param bool   $has_template  True if the template is available.
-	 * @param string $template_name The name of the template.
-	 *
-	 * @return bool True if the system is checking archive-product
-	 */
-	public function has_block_template( $has_template, $template_name ) {
-		if ( 'archive-product' === $template_name || 'taxonomy-product_brand' === $template_name ) {
-			$has_template = true;
-		}
-
-		return $has_template;
-	}
-
-	/**
-	 * Get the block template for Taxonomy Product Brand. First it attempts to load the last version from DB
-	 * Otherwise it loads the file based template.
-	 *
-	 * @param string $template_type The post_type for the template. Normally wp_template or wp_template_part.
-	 *
-	 * @return WP_Block_Template The taxonomy-product_brand template.
-	 */
-	private function get_product_brands_template( $template_type ) {
-		$template_db = $this->get_product_brand_template_db();
-
-		if ( $template_db ) {
-			return BlockTemplateUtilsDuplicated::gutenberg_build_template_result_from_post( $template_db );
```

Comment:
I think now the `BlockTemplateUtilsDuplicated` class can be removed? I believe it wasn't used anywhere else.